.github/workflows/lint_python.yml

```diff
@@ -1,30 +0,0 @@
-name: lint Python
-on: [pull_request]
-jobs:
-  lint_python:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-      - uses: pre-commit/action@v3.0.0
-      - run: pip install --upgrade pip wheel
-      - run: pip install -U bandit codespell flake8 isort pytest pyupgrade tox
-
-      - name: bandit
-        run: bandit -r . || true
-
-      - name: Run CodeSpell
-        run: codespell --ignore-words-list="brane,gool,ist,sherif,wil" --quiet-level=2 --skip="*.key" || true
```

Comment:
This is a waste of resources. No one checks the issues.

I've fixed the issues from codespell on https://github.com/celery/celery/pull/7815.